src/lib/structures/game/StandardGame.ts

```diff
@@ -120,6 +121,22 @@ export class StandardGame extends Game {
 		await this.textChannel.send({ embeds: [embed] });
 	}
 
+	public validGuessChannel(channel: TextChannel | DMChannel | PartialDMChannel) {
+		if (!isTextChannel(channel)) {
+			return false;
+		}
```

Comment:
This is useless, the ID wouldn't match anyway

Reply:
Without it the linter complains on accessing channel.guild in the next line:
`Property 'guild' does not exist on type 'TextChannel | DMChannel | PartialDMChannel'.
  Property 'guild' does not exist on type 'DMChannel'.ts(2339)`

Reply:
I would cast there, also the type of `channel` could be `TextBasedChannels` to save a few imports